src/components/MapboxMap/MapboxMap.vue

```diff
@@ -107,54 +112,134 @@ function initMap(accessToken: string) {
     });
   }
 
+  function addBuildingOccupancy(map: mapboxgl.Map, markerName: string) {
+    return new Promise<void>((resolve) => {
+      const img = new Image(145, 33);
+      img.onload = () => {
+        map.addImage(markerName, img);
+        resolve();
+      };
+      img.src = `/icons/clusters/${markerName}.svg`;
+    });
+  }
+
   map.on("load", () => {
     const markerNames = [...OCCUPANCY_RATES, "unknown"] as const;
-    Promise.all(
-      markerNames.map((occupancy) => addMarker(map, `map-marker-${occupancy}`))
-    ).then(() => {
+    Promise.all([
+      markerNames.map((occupancy) => {
+        addMarker(map, `map-marker-${occupancy}`)
```

Comment:
Beforehand the marker name was mapped to a promise of adding an image and we waited for all those promises to resolve before continuing with `then`. Now we add two images per marker name, but don't wait for any corresponding promise to resolve, continuing immediately because the code in the curly braces returns void without waiting. Is it OK? Does it work that way without any issues? Then I guess to explicitly state that we better just `for..of` the occupancy values instead of `map` (which should return values) and not do any `Promise-all-then`. If we want to keep the previous flow, we'd have to do something :) Like await for every image add call or something like that.